packages/common/storage/src/typed/storages/NumberTypedStorage.ts

```diff
@@ -2,13 +2,13 @@
 import { TypedStorage } from './TypedStorage';
 
 export class NumberTypedStorage extends TypedStorage<number> {
-  public increase(): void {
+  public increase(offset = 1): void {
```

Comment:
Please specify the type of `offset`.
```typescript
public increase(offset: number = 1): void;
```

Reply:
It's good that it's explicit! but it is ok to ignore line warning?

![image](https://user-images.githubusercontent.com/69495129/216388924-9573c3fc-9bf3-4b76-90fd-b7bbcfd71505.png)